crates/iota-e2e-tests/tests/reconfiguration_tests.rs

```diff
@@ -876,12 +870,12 @@ async fn safe_mode_reconfig_test() {
     assert!(system_state.epoch_start_timestamp_ms() >= prev_epoch_start_timestamp + EPOCH_DURATION);
 
     // Try a staking transaction.
-    let validator_address = match system_state.into_iota_system_state_summary() {
-        IotaSystemStateSummary::V1(v1) => v1.active_validators,
-        IotaSystemStateSummary::V2(v2) => v2.active_validators,
-        _ => panic!("unsupported IotaSystemStateSummary"),
-    }[0]
-    .iota_address;
+    let validator_address = system_state
```

Comment:
```suggestion
    let comittee_member_address = system_state
```

Reply:
Thanks, addressed in [5cffc30](https://github.com/iotaledger/iota/pull/6015/commits/5cffc301687114db6f95c90c2bc1c5c238f23c48).